runtime/moonbeam/src/lib.rs

```diff
@@ -115,6 +115,21 @@ pub fn native_version() -> NativeVersion {
 
 const NORMAL_DISPATCH_RATIO: Perbill = Perbill::from_percent(75);
 
+/// Returns if calls are allowed through the filter
+pub struct BaseFilter;
+impl Filter<Call> for BaseFilter {
+	fn filter(c: &Call) -> bool {
+		match c {
+			Call::Balances(_) => false,
+			Call::CrowdloanRewards(_) => false,
+			Call::Democracy(_) => false,
+			Call::Ethereum(_) => false,
+			Call::EVM(_) => false,
+			_ => true,
+		}
+	}
+}
+
```

Comment:
Just to be sure, current callable pallets that are not filtered:

- System
- Utility
- Timestamp
- RandomnessCollectiveFlip
- ParachainSystem
- ParachainStaking
- Scheduler
- CouncilCollective
- TechComitteeCollective
- AuthorInherent

If filter does not apply to sudoer, shouldn't we filter all the above in stage 0?

Reply:
We want the foundation to be able to stake in stage 0 so ParachainStaking is not filtered. I don't know for the others, what do you think @crystalin

Reply:
> If filter does not apply to sudoer, shouldn't we filter all the above in stage 0?

Why does the filter not apply to the sudoer? I'm going to look further into how this BaseCallFilter is used in frame_system now.

Reply:
Yeah, I'm not sure either. Was reading the code around and saw:
https://github.com/paritytech/substrate/blob/c93ef27486e5f14696e5b6d36edafea7936edbc8/frame/support/src/origin.rs#L263

and made me wonder if root origin is not filtered (which would make sense).

Reply:
I think we shouldn't filter them unless we need to. I prefer minimizing changes until we test this feature more.

Reply:
If we decide there is danger in not filtering any of those listed above then we can add them in a follow-up